projects/components/src/datagrid/datagrid.component.spec.ts

```diff
@@ -110,6 +111,33 @@ describe('DatagridComponent', () => {
                 );
             });
 
+            describe('@Input() columns.disableCliptext', () => {
+                beforeEach(function(this: HasFinderAndGrid): void {
+                    this.finder.hostComponent.columns = [
```

Comment:
You can probably configure this in each `it` separately. Then you don't need magic indexes in each test